truffle/src/com.oracle.truffle.dsl.processor/src/com/oracle/truffle/dsl/processor/parser/NodeParser.java

```diff
@@ -1748,6 +1749,13 @@ private void collectIncludes(SpecializationData specialization, Set<Specializati
         }
     }
 
+    private static void initializeUncached(final NodeData node) {
```

Comment:
I think this could be merged into initializeReachability. It is fine if this one computes cached and uncached reachability.

Reply:
Again, the `intializeReachability` methods computes both kinds of reachability. This one only initializes a helper field.